src/tools/batchrun/main.cpp

```diff
@@ -230,7 +230,7 @@ int main(int argc, char* argv[])
 
         // The folder that contains the solver
         String dirname;
-        IO::ExtractFilePath(dirname, solver);
+        IO::parent_path(dirname, solver);
```

Comment:
```suggestion
        IO::ParentPath(dirname, solver);
```

Reply:
I used snake case to match std https://en.cppreference.com/w/cpp/filesystem/path/parent_path

Reply:
I think we want to use CamelCase in Antares Simulator.